__tests__/get.test.js

```diff
@@ -90,166 +90,181 @@ describe("/api/articles/:article_id", () => {
 });
 
 describe("GET /api/articles", () => {
-	test("GET:200 articles have the correct properties and data types respectively", () => {
-		return request(app)
-			.get("/api/articles")
-			.expect(200)
-			.then(({ body }) => {
-				expect(body.articles[0]).toEqual(
-					expect.objectContaining({
-						title: expect.any(String),
-						topic: expect.any(String),
-						author: expect.any(String),
-						article_id: expect.any(Number),
-						created_at: expect.any(String),
-						votes: expect.any(Number),
-						article_img_url: expect.any(String),
-						comment_count: expect.any(Number),
-					})
-				);
-			});
-	});
-	test("GET:200 sends an array of all articles with the correct properties", () => {
-		return request(app)
-			.get("/api/articles")
-			.expect(200)
-			.then(({ body }) => {
-				expect(Array.isArray(body.articles)).toBe(true);
-				expect(body.articles.length).toBeGreaterThan(0);
-				body.articles.forEach((article) => {
-					expect(article).toMatchObject({
-						author: expect.any(String),
-						title: expect.any(String),
-						article_id: expect.any(Number),
-						topic: expect.any(String),
-						created_at: expect.any(String),
-						votes: expect.any(Number),
-						article_img_url: expect.any(String),
-						comment_count: expect.any(Number),
+	describe("GET /api/articles core functionality ", () => {
+		test("GET:200 articles have the correct properties and data types respectively", () => {
+			return request(app)
+				.get("/api/articles")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body.articles[0]).toEqual(
+						expect.objectContaining({
+							title: expect.any(String),
+							topic: expect.any(String),
+							author: expect.any(String),
+							article_id: expect.any(Number),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							article_img_url: expect.any(String),
+							comment_count: expect.any(Number),
+						})
+					);
```

Comment:
In this test as you are using the article at index `[0]` you could use the exact values for the properties in this test as they are in the test data 🤔